pgjdbc/pom.xml

```diff
@@ -36,6 +36,14 @@
     <checkstyle.version>7.8.2</checkstyle.version>
   </properties>
 
+  <dependencies>
+    <dependency>
+      <groupId>com.ongres.scram</groupId>
+      <artifactId>client</artifactId>
+      <version>1.0.0-beta.2</version>
```

Comment:
Will this be  a `beta` dependency?

Reply:
That bothers me a bit as well.

Reply:
I was planning on bumping it to 1.0.0 when it is well tested and ready to be integrated. So if you feel this is OK, I will proceed and release 1.0.0 and update the PR.

Reply:
@ahachete well, it will never be tested if it's not included in pgjdbc, so is a chicken-egg problem, it's your code, so you better know if it is well tested.

I'm Ok if it's released as a *Beta* feature, mark it as such in release notes and if everything goes fine, bumping it to 1.0.0 and update it for the next release cycle.